packages/core/src/browser/saveable.ts

```diff
@@ -179,6 +179,10 @@ export namespace SaveableWidget {
     }
 }
 
+export interface SaveOptions {
```

Comment:
Although there's only one option at the moment, it might be good to anticipate adding more later and make the fields optional, so that we think about defaults from the start for cases where only partial options are specified. In this case, since the default is `false`, I don't think anything would be required other than adding the `?` here.

Reply:
I was thinking the same, there is a new use-case where we want to use the `options` as well that I spoke with @DukeNgn about offline. The new property would be to distinguish when we are performing a `save` versus a `save all`.

Using `optional` properties would be a good idea :+1:

Reply:
@colin-grant-work @vince-fugnitto 
Thank you. That's actually a good idea. I'll update the PR.